server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java

```diff
@@ -172,7 +172,8 @@ private ExtensionVersion findExtensionVersion(String namespace, String extension
     @Override
     public ResponseEntity<byte[]> getFile(String namespace, String extensionName, String targetPlatform, String version, String fileName) {
         var extVersion = findExtensionVersion(namespace, extensionName, targetPlatform, version);
-        var resource = repositories.findFileByName(extVersion, fileName);
+        var resource = isType(fileName) ? repositories.findFileByType(extVersion, fileType(fileName)) : repositories.findFileByName(extVersion, fileName);
+        repositories.findFileByName(extVersion, fileName);;   
```

Comment:
You can remove this line.

Reply:
Thank you for reviewing and giving these comments. May I ask why we should remove this line? How would the `resource` variable exist in this case? Thanks!

Reply:
I mean line 176: `repositories.findFileByName(extVersion, fileName);;` 
This extra query is not assigned to a variable.